cassandra/install.sh

```diff
@@ -9,6 +12,19 @@ mv apache-cassandra-$CASSANDRA_VERSION/* /cassandra/
 echo "*** Installing Python"
 apk add --update --no-cache python
 
+# Default conf for Cassandra 3.x does not work on modern JVMs due to many deprecated flags
```

Comment:
@openzipkin/cassandra PTAL noting this is a demo image and not intended for prod